ncm-metaconfig/src/main/metaconfig/icinga-web/databases.tt

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ae:configurations xmlns:ae="http://agavi.org/agavi/config/global/envelope/1.0" xmlns="http://agavi.org/agavi/config/parts/databases/1.0">
+    <ae:configuration>
+        <databases default="icinga_web">
+[%-         INCLUDE "metaconfig/icinga-web/database.tt" data=CCM.contents FILTER indent FILTER indent FILTER indent -%]
```

Comment:
filter indent sponatenous multiplication? why not indent each section instead of using 4 spaces?